packages/firebase_analytics/ios/Classes/FirebaseAnalyticsPlugin.m

```diff
@@ -57,7 +57,7 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
     result(nil);
   } else if ([@"setAnalyticsCollectionEnabled" isEqualToString:call.method]) {
     NSNumber *enabled = [NSNumber numberWithBool:call.arguments];
-    [[FIRAnalyticsConfiguration sharedInstance] setAnalyticsCollectionEnabled:[enabled boolValue]];
+    [FIRAnalytics setAnalyticsCollectionEnabled:[enabled boolValue]];
```

Comment:
Does this method exist in the previous pod version? It would be a breaking change, if no.

Reply:
Looks like it was moved here fairly recently.